### PR TITLE
Dust off the roller beds

### PR DESCRIPTION
**Pull request by aceedex**
This adds a new reagent, Lectulolin. It heals all damage by 3 every tick if the person is in critical, and heals for another 3 if that person is buckled to something. It will also last longer on a person thats buckled. For reference thats sightly better than cryo/clonex if the person is buckled. Its made with doctor's delight, synthocarisol, peptobismol and dermaline as well as having plasma and dexalin plus as catalysts. Will probably add this to paramedics somehow should it go through

Disclaimer: Ive never used an else clause in my life

=
:cl:
 * rscadd: adds lectulolin. A reagent that heals massive ammounts quickly provided the holder is in critical and buckled to something

### CODE REVIEW

File: code/modules/organs/organ_external.dm [hunk 1]
```diff
@@ -1150,10 +1150,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 		var/emote_scream = pick("screams in pain and", "lets out a sharp cry and", "cries out and")
 		owner.emote("me", 1, "[owner.feels_pain() ? emote_scream : ""] drops what they were holding in their [hand_name]!")
 	if(is_malfunctioning())
-		// owner.u_equip(c_hand, 1)
-		owner.emote("me", 1, "drops what they were holding, their [hand_name] malfunctioning!")
-		spark(src, 5, FALSE)
-		owner.drop_item(c_hand)
+		return (is_robotic() && (status & (ORGAN_MALFUNCTIONING) || (brute_dam + burn_dam > min_broken_damage/2 && prob(brute_dam + burn_dam))))
```
Comment (MadmanMartian) [inline code/modules/organs/organ_external.dm]: ???

Comment (aceedex) [reply code/modules/organs/organ_external.dm]: I forgot to unfuck stuff from previous PRs and am paying the price

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
It's kind of really complex and OP
If your aim was to make rollerbeds more common, I would personally go for something simple and ubiquitous. Something like "inaprovaline has a secondary bonus on rollerbed'd patients"

**Comment by Kurfursten** [comment]
It feels very creep to make a chemical that heals !!!3X!!! as fast as Doctor's Delight and stacks with it, along with everything else.

I realize that you've gated it behind so much chemistry that it comes just short of requiring a vision quest but still.

**Comment by aceedex** [comment]
yeah I realize it might seem very strong but a lot of its utility is linked to the bed application so I dont see how this could break anything. It doesnt have much in-combat use and its no more offensive to normal medbay than the magic heal tube. Still I wouldnt mind moving more of it's application to the buckled part if thats a worry. Wouldn't mind making it remove all reagants or something if it pulls you out of crit to prevent stacking memes but I dont think its an issue

And to mr. cloakaman, the idea is to provide it to the paramed roundstart as to make people use roller beds and make their job less corpse recovery. I really dont think anything short of damage stasis will entice people to waste time with a roller bed. I did keep the chem requirements high so that its not exclusive/can be refilled if needed.

If you do think its all a bit much, Im all ears to suggestions

**Comment by DamianX** [comment]
This is the ultimate memechair chemical. Gets a thinking emoji from me :thinking:

**Comment by codeungabunga** [comment]
Bad recipe that will never be used.

**Comment by Caboozles** [comment]
If you don’t think this’ll be used you’re dead wrong. But not for the reasons we’d want.

This functions more like a combat drug. You go into crit? You’re basically immediately out unless you have major blood loss with it. Especially with no OD (I think) and the ability to chug it en masse.
Not the way to go.

**Comment by aceedex** [comment]
Nah fam its hard  to make to prevent it from being laced to every other pill in existence

**Comment by Tokanova** [comment]
do the thing where it removes other chems when it takes you out of crit and i think this could work. maybe make it remove itself too?

**Comment by Caboozles** [comment]
Hard to make means nothing to a bored chemist or a powergamer with a plan.
A quick snag of the ghetto machines, a quick snag of a groans, and maybe 15 minutes could net you a few hundred units easily.
If it did the above and removed itself it’d be more tolerable but still wouldn’t be beneficial enough to make, except as a self-emergency pill.

Just give this feature to inapprov as long as someone is tied to a bed.

**Comment by bathosbathtime** [comment]
I would say that it is almost better off not having a recipe at all if you only want paramedics using it.
The only use for this that I personally can think of is cyro fodder.

**Comment by Blithering** [comment]
maybe have roller beds able to dispense inaprovaline instead. Also if bucked to something counts wheelchairs hello 50% of murderchair chemist's chempack content

**Comment by aceedex** [comment]
🙏 appreciate the feedback. Ill close this for now and will probably make a new PR with the effects added to inaprovalin, with the effects taking place ONLY if the person is buckled and in critical, as well as adding a memechair check to prevent abuse. Will probably lower the healing a lil bit since inaprov is super easy to get. Thanks for the ideas

**Pull request closed without merging**